lib/src/features/devices/presentation/training/statistic/precision/wall_statistic.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:zone_app/main.dart';
+
+class WallStatistic extends StatefulWidget {
+  Color borderColor;
+  int hits;
+
+  WallStatistic(
+      {Key? key, this.borderColor = ZoneTheme.redColor, this.hits = 0})
+      : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() {
+    return WallStatisticState();
+  }
+}
+
+class WallStatisticState extends State<WallStatistic> {
+  @override
+  Widget build(BuildContext context) {
+    return Flexible(
+      flex: 3,
+      fit: FlexFit.loose,
+      child: SizedBox(
+        height: MediaQuery.of(context).size.height * 0.1,
```

Comment:
Please, make it a little beat less high
```suggestion
        height: MediaQuery.of(context).size.height * 0.09,
```